style.css

```
@@ -950,6 +968,173 @@ body.dark-mode .icon-sun {
+.detail-locate-action:hover {
+  border-color: var(--accent-hover);
+  background: var(--accent-hover);
+  box-shadow: var(--shadow-sm);
+  transform: translateY(-1px);
+}
+
+.detail-locate-icon {
+  width: 17px;
+  height: 17px;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)